addons/website_slides/models/slide_channel.py

```diff
@@ -191,6 +203,11 @@ def _init_column(self, column_name):
 
     @api.model
     def create(self, vals):
+        if vals.get('visibility') == 'invite' and not vals.get('channel_partner_ids'):
```

Comment:
I guess that's meant to add the user creating the channel as member of it?
Any particular reason? Maybe add a small comment?

Reply:
Indeed. Just to be sure there is at least someone in it. I will add a comment.